x-pack/plugins/security_solution/server/lib/detection_engine/signals/threat_mapping/create_threat_signals.ts

```diff
@@ -57,6 +57,7 @@ export const createThreatSignals = async ({
 
   let results: SearchAfterAndBulkCreateReturnType = {
     success: true,
+    warning: false,
     bulkCreateTimes: [],
     searchAfterTimes: [],
     lastLookBackDate: null,
```

Comment:
Could we initialize this with the same `createSearchAfterReturnType()` function used in the other executors?

Reply:
We can, I just didn't do it in this PR to limit scope creep